tests/api_tests/base/test_serializers.py

```diff
@@ -50,3 +53,46 @@ def test_invalid_param_raises_bad_request(self):
 
         res = self.app.get(self.url, params={'related_counts': 'fish'}, expect_errors=True)
         assert_equal(res.status_code, http.BAD_REQUEST)
+
+
+class TestJSONAPIHyperlinkedIdentityField(DbTestCase):
```

Comment:
This test class does not need a Webtest app; it only needs a scratch database. `make_drf_request` is necessary because `HyperlinkedIdentityField` requires a `request` in its serializer's context.